lib/rspec_controller_helpers/resourceful.rb

```diff
@@ -1,23 +1,43 @@
 RSpec.shared_examples_for 'a resourceful controller' do |model, raw_options|
-  let(:options) { raw_options || {} }
```

Comment:
Might be a good idea to have `default_options` set at the top for
1) Other devs/users to see what options are available without searching throughout.
2) To avoid safe guarding below (e.g. options[:x] || [])